crates/cxx-qt-gen/src/generator/cpp/qobject.rs

```diff
@@ -126,7 +127,7 @@ impl GeneratedCppQObject {
                 // CODECOV_EXCLUDE_STOP
             }
         });
-        generated.blocks.base_classes.push(base_class.clone());
+        generated.blocks.base_classes.push(base_class.to_string());
```

Comment:
i think when this is not using the fallback of a QObject and the user has specified something. We need to do the lookup a type_names.lookup with it and then find the `cxx_qualified()`. As we might need the string to be `namespace::Base` or it might have a cxx_name etc